src/Restaurant.js

```diff
@@ -1,14 +1,12 @@
 import React from "react";
+import Order from "./order";
 
 const Restaurant = () => {
-  const pizzas = 0;
   return (
-    <div>
+    <div className="orders">
       <h3>Restaurant Orders</h3>
       <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
+        <Order />
```

Comment:
This looks good, but exercise 13 asks you to render some additional `Order` components:

> render 2 others <Order /> components but this time pass different values for the orderType prop: "Salads" and "Chocolate cake".

Could you make those changes too?

Reply:
@40thieves  I didn't see  this, I fixed it